### ISSUE TITLE
Cameras not being detected

### ISSUE THREAD

**Issue by mcinnes01** [briis/unifiprotect#83]
I'm on the latest home assistant 0.111.2 and I have a UDM pro latest version 1.7.2rc5 latest Unifi and latest protect on the UDM pro. I just got a g3 flex and followed your instructions creating a local user for the UDM pro, enabling medium rtsp, port 443, etc. But in the integration I get one device which is the UDM pro, but no camera and no entities. Any ideas where I might be going wrong?

**Comment by briis** [comment]
This is a new one. So if you go in to the *Developer* Tab and select *States* you don't see a Camera entity in there? What have you named your Camera in Unifi Protect? I was just wondering if there is a special character that makes it fail?

Try and add the following to your `configuration.yaml`, reboot and then look in the log file for some errors:

````
logger:
  default: error
  logs:
    custom_components.unifiprotect: debug
````

**Comment by briis** [comment]
Hi @mcinnes01 
I assume this is resolved, else please re-open the issue, and supply more information.

**Comment by mcinnes01** [comment]
Hi 

Sorry been consumed by other things of late. The camera is called "Basement Hall Camera" no special characters.

I've just added the logs will check now and post results back...

**Comment by to4ko** [comment]
same here...only one string that data fetched in 4secs and that's all....no cameras, no motion sensors ... nothing....

**Comment by iamandymcinnes** [comment]
I've tried all sorts, I'm on latest beta firmware on my udm pro, latest protect, latest unifi, latest beta of home assistant and I've readded and removed the integration to no avail. My camera name has only letters and spaces no other characters.

**Comment by to4ko** [comment]
same here - no special characters in camera name
protect ver 1.14.10
udm pro firmware 1.7.2


will keep using unifi video....

**Comment by to4ko** [comment]
hm...looks like i have something - i added Protect application and Admin rights to my local user (under which i was logging in integration). Now it gets my cam did a switches and sensors. but i'm unable to get any footage in HA

**Comment by iamandymcinnes** [comment]
very interesting, I'll check my permissions now

**Comment by iamandymcinnes** [comment]
So the udm local user has limited admin access and I've now granted administrator access against protect (which I was missing). I deleted and re-added the protect integration and now have an empty udm device and my camera. I can see my camera, but the stream took a couple of moments to display. I think if I remember I have the medium quality rtsp stream enabled on the camera. Perhaps for home assistant purposes the low quality stream is sufficient. Have you enabled an RTSP stream?
